example/async-stream-server.rs

```diff
@@ -152,6 +152,25 @@ impl streaming_ttrpc::Streaming for StreamingService {
 
         Ok(())
     }
+
+    async fn server_send_stream(
+        &self,
+        _ctx: &::ttrpc::r#async::TtrpcContext,
+        _: empty::Empty,
+        s: ::ttrpc::r#async::ServerStreamSender<streaming::EchoPayload>,
+    ) -> ::ttrpc::Result<()> {
+        let mut seq = 0;
+        while seq < 10 {
+            sleep(std::time::Duration::from_secs(1)).await;
```

Comment:
sleep 1 sec is too long to an example in my opinion. How about using 100ms instead? thanks.

Reply:
Updated.